lib/rules/no-get.js

```diff
@@ -193,7 +206,7 @@ module.exports = {
 
         if (
           types.isIdentifier(node.callee) &&
-          node.callee.name === 'get' &&
+          node.callee.name === importedGetName &&
```

Comment:
It's possible that the imported get name is shadowed by something else in a more local scope. Is there a good way to detect that?

Reply:
That's a good edge case. Not sure of a convenient way to detect that at the moment.

Reply:
This rule detects that by the way: https://eslint.org/docs/rules/no-shadow